control_protocol.md

```diff
@@ -330,17 +330,490 @@ A Component MUST also offer a list of all possibly callable methods in accordanc
+        },
+        {
+            "name": "coordinator_sign_in",
+            "params": [],
+            "result": {
+                "name": "result",
+                "schema": {
+                    "type": "null"
+                },
+                "required": true
+            }
+        },
+        {
+            "name": "coordinator_sign_out",
+            "params": [],
+            "result": {
+                "name": "result",
+                "schema": {
+                    "type": "null"
+                },
+                "required": true
+            }
+        },
+        {
+            "name": "set_nodes",
+            "params": [
+                {
+                    "name": "nodes",
+                    "schema": {
+                        "type": "object",
+                        "additionalProperties": true
+                    },
+                    "required": true
+                }
+            ],
+            "result": {
+                "name": "result",
+                "schema": {
+                    "type": "null"
+                },
+                "required": true
+            }
+        },
+        {
+            "name": "set_remote_components",
+            "params": [
+                {
+                    "name": "components",
+                    "schema": {
+                        "type": "array",
+                        "items": {
+                            "type": "string"
+                        }
+                    },
+                    "required": true
+                }
+            ],
+            "result": {
+                "name": "result",
+                "schema": {
+                    "type": "null"
+                },
+                "required": true
+            }
+        },
+        {
+            "name": "compose_global_directory",
+            "params": [],
+            "result": {
+                "name": "result",
+                "schema": {
+                    "type": "object",
+                    "additionalProperties": true
+                },
+                "required": true
+            }
+        },
+        {
+            "name": "compose_local_directory",
+            "params": [],
+            "result": {
+                "name": "result",
+                "schema": {
+                    "type": "object",
+                    "additionalProperties": true
+                },
+                "required": true
+            },
+            "description": "Compose a dictionary with the local directory."
+        },
+        {
+            "name": "get_component_names",
+            "params": [],
+            "result": {
+                "name": "result",
+                "schema": {
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "required": true
+            }
+        },
+        {
+            "name": "clean_addresses",
```

Comment:
"clean" could also be understood "delete all" IMO. Maybe `prune_addresses`, thinking of `git fetch --prune`, or something similar -- `remove_expired_addresses`?

Reply:
Done.